src/main/service/createAccountService.ts

```diff
@@ -32,48 +34,53 @@ const accountManagementRequests = new AccountManagementRequests();
 const fileHandlingService = new FileHandlingService();
 
 export class CreateAccountService {
-  public validateFormFields(formValues: object, file: File): object {
+  public validateFormFields(formValues: object, file: File,
+    language: string, languageFile: string): object {
     return {
       nameError: {
-        message: this.validateMediaFullName(formValues['fullName']),
+        message: this.validateMediaFullName(formValues['fullName'], language, languageFile),
```

Comment:
When getting the text for the error within each of the validation method, you are calling the language parser getText() method. This will read from the file every time. I think it might be better to read the json from the file once at the beginning of this method, pass down the json into each validation method, and create another method for getText() in the language parser which does not involves reading from file every time.

Reply:
agreed... and fixed the code